README.md

```diff
@@ -46,15 +45,13 @@ git clone -b v17.9.1 --single-branch git@github.com:GEOS-ESM/GEOSldas.git
 To build the model in a single step, do the following from a head node:
 ```
 cd ./GEOSldas
-parallel_build.csh
+./parallel_build.csh -mil
```

Comment:
I guess initially we will keep using `-mil`, but looking ahead, do we really need to have the `-mil` flag for parallel_build.csh if there's only SLES15?

Reply:
we will see how build.csh handle the default.  @mathomp4